Tests/FunctionalTests/helpermethod.ts

```diff
@@ -147,3 +156,62 @@ export async function AcceptCookies(page: Page) {
     }
   }
 }
+
+export async function ExpectAllResultsHaveBatchUserAttValue(
+  page: Page, preciseValue: string): Promise<number> {
+
+  return await ExpectSelectionsAreEqual(page,
+    `//table[@class='${pageObjectsConfig.searchAttributeTable.substring(1)}']`,
+    `//table[@class='${pageObjectsConfig.searchAttributeTable.substring(1)}' and 0 < count(.//td[translate(text(),'ABCDEFGHIJKLMNOPQRSTUVWXYZ', 'abcdefghijklmnopqrstuvwxyz')='${preciseValue.toLowerCase()}'])]`);
+}
+
+export async function ExpectAllResultsContainBatchUserAttValue(
```

Comment:
It is unexpected for methods named `Expect` to return a value. Please can all of these methods be amended so that they are only responsible for asserting things and not for returning a resultcount

Reply:
New method GetCountOfBatchRows() added and return value removed